src/Fixer/ControlStructure/NoUselessElseFixer.php

```diff
@@ -198,6 +229,35 @@ private function fixEmptyElse(Tokens $tokens, $index)
      */
     private function clearElse(Tokens $tokens, $index)
```

Comment:
This method name is misleading now, as it does not just clear the `else` statement anymore.
Please refactor the logic into a new private method and leave this one as is.